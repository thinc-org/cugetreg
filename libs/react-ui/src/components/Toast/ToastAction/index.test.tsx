import { shallow } from 'enzyme'

describe('ToastAction', () => {
  const mockChildren = <div>children</div>

  it('Should render correctly', async () => {
    const { ToastAction } = await import('.')
    const wrapper = shallow(<ToastAction>{mockChildren}</ToastAction>)

    expect(wrapper).toMatchSnapshot()
  })
})
