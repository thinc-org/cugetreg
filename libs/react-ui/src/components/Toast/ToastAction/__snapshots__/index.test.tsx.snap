// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ToastAction Should render correctly 1`] = `
<Fragment>
  <Insertion
    cache={
      Object {
        "insert": [Function],
        "inserted": Object {},
        "key": "css",
        "nonce": undefined,
        "registered": Object {},
        "sheet": StyleSheet {
          "_insertTag": [Function],
          "before": null,
          "container": <head />,
          "ctr": 0,
          "insertionPoint": undefined,
          "isSpeedy": false,
          "key": "css",
          "nonce": undefined,
          "prepend": undefined,
          "tags": Array [],
        },
      }
    }
    isStringTag={false}
    serialized={
      Object {
        "map": undefined,
        "name": "3o8730",
        "next": undefined,
        "styles": "
  min-width: 0;
  height: 100%;
  padding: 0 16px;
  border: none;
  border-left: 1px solid rgba(0, 0, 0, 0.12);
  border-radius: 0;

  &:last-child {
    border-radius: 0 3px 3px 0;
  }
",
        "toString": [Function],
      }
    }
  />
  <ForwardRef(Button)
    className="css-3o8730"
  >
    <div>
      children
    </div>
  </ForwardRef(Button)>
</Fragment>
`;
